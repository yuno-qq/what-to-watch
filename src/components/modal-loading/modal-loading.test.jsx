import React from "react"
import {describe, test, expect} from "@jest/globals"
import renderer from "react-test-renderer"

import ModalLoading from "./modal-loading.jsx"


describe(`<ModalLoading> snapshots:`, () => {
  test(`should render correctly`, () => {
    const tree = renderer.create(<ModalLoading />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
