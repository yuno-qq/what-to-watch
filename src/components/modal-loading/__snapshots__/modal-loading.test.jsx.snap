// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModalLoading> snapshots: should render correctly 1`] = `
<div
  className="modal"
>
  <div
    className="modal__content"
  >
    <h2>
      Идет загрузка...
    </h2>
    <p>
      Пожалуйста, подождите
    </p>
  </div>
</div>
`;
